third_party/binutils/test_decenc/test_decenc_x86.asm

```diff
@@ -311,13 +311,6 @@ GLOBAL_LABEL(FUNCNAME:)
         /* xsaves, CPL 0 instruction. */
         /* RAW(0f) RAW(c7) RAW(29) */
         RAW(66) RAW(0f) RAW(ae) RAW(39)
-        /* FIXME i3577: monitorx, AMD only. */
```

Comment:
These are grouped by the binutil test file name: see the comment at line 299.  They should be kept together. 
 Please move the whole group for that file as one.